rules/autoconsent/livejasmin.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "livejasmin",
+  "vendorUrl": "https://www.livejasmin.com/",
+  "runContext": {
+    "urlPattern": "^https://www\\.livejasmin\\.com/"
+  },
+  "prehideSelectors": ["#consent_modal"],
+  "detectCmp": [
+    {
+      "exists": "#consent_modal"
+    }
+  ],
+  "detectPopup": [
+    {
+      "visible": "#consent_modal"
+    }
+  ],
+  "optIn": [
+    {
+      "waitForThenClick": "#consent_modal button.jOyTQg"
+    }
+  ],
+  "optOut": [
+    {
+      "waitForThenClick": "#consent_modal button.bwexRL"
```

Comment:
This opt-out didn't work for me. It looks like the button has a different class. Randomized CSS classes again?
![Screenshot 2024-02-01 at 12 10 32](https://github.com/duckduckgo/autoconsent/assets/248111/360daa00-1f6f-469b-abb5-88095719ca7f)